index.html

```diff
@@ -5,9 +5,9 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Mobile Version Skeleton</title>
-    <link rel="stylesheet" href="style.css">
-    <link rel="stylesheet" href="desk.css">
+    <title>Personal Portfolio</title>
```

Comment:
- [ ]  Kindly update title in the head to briefly describe the content of your page e.g; Christabelle's portfolio or Christabelle full-stack webdeveloper

Reply:
Hello @prolajumokeoni 
My partner and I added links to the corresponding line of codes which had issues and were modified.
Please you can check the above at the level of the issues found each point has a link to the issue in the codes

Reply:
Hey @prolajumokeoni 
Don't know if you meant another type of link

Reply:
hey... @prolajumokeoni